### ISSUE TITLE
Multiple repls at once, using tabs

### ISSUE THREAD

**Issue by spotandjake** [repl-it-discord/repl-it-electron#107]

**Comment by frissyn** [comment]
@spotandjake 

This is pretty redundant since we're also gonna add multi-window support. Couldn't you just open a new repl in a new win?

**Comment by spotandjake** [comment]
Not really if you are using single monitor, the tabs really help that is main reason i stopped using the app tbh i think by multimonitor support we are more talking your logs on one page that sort of thing

**Comment by frissyn** [comment]
Ok sure

**Comment by whippingdot** [comment]
Wait we NEED THIS. Whenever I open in something in a new tab it takes me to a new window. I would like it if it had a design a little like Taskade (though their multi-tab thing looks horrible). This isn't major though, as I am ok with it opening in a new window (but I would prefer not to as I like having the minimum number of windows open possible)

**Comment by spotandjake** [comment]
Yes lol that is why it is here . But it is gonna require some major rework so I do not know how soon

**Comment by whippingdot** [comment]
Eh i got the dev thing branch so i wud get the updates first. LES GOOO. I don't care how long it takes though as this seems REALLY complicated

**Comment by whippingdot** [comment]
Oops sorry @spotandjake but this has to be reported. When you open the link in a new window it doesn't use the same theme too. It just has the default dark theme/light theme of repl.it.

**Comment by spotandjake** [comment]
I will have to do some more testing dev isn't done though so it is prerelease expect bugs if you are using. It

**Comment by whippingdot** [comment]
I know i am just reporting the issue

**Comment by JDJGInc** [comment]
I think having multiple tab support and multi window support would be useful.

**Comment by spotandjake** [comment]
Yah we want to do this the issue is working with ui in a reliable way is a little tricky. we may push this to v4 though so hopefully it makes it into v3.
